src/util/std_expr.h

```diff
@@ -63,9 +52,8 @@ inline const transt &to_trans_expr(const exprt &expr)
   return static_cast<const transt &>(expr);
 }
 
-/*! \copydoc to_trans(const exprt &)
- * \ingroup gr_std_expr
-*/
+/// \copydoc to_trans(const exprt &)
+/// \ingroup gr_std_expr
```

Comment:
Is this \ingroup useful?

Reply:
It might have been to avoid reference conflicts when using `\copydoc`, but I don't see where, so I've removed `\ingroup` everyhwere.